demos/calendar/dropdown-month-year.html

```diff
@@ -8,18 +8,81 @@
 	<link rel="stylesheet" href="../demos.css">
 	<script src="../../external/requirejs/require.js"></script>
 	<script src="../bootstrap.js">
-		$( "#calendar" ).calendar({
-			changeMonth: true,
-			changeYear: true
+		$.widget( "custom.calendarWithSelect", $.ui.calendar, {
+			_buildTitle: function() {
+				var title = $( "<div>", { role: "alert", id: this.gridId + "-month-label" } ),
```

Comment:
Hmm...Can we come up with a way to avoid using internal properties like `this.gridId`?